submissions/xT0PERx/html-movie-seat-booking/main.css

```diff
@@ -0,0 +1,350 @@
+  grid-template-columns: repeat(5, 1fr);
+  grid-column: span 5;
+  list-style-type: none;
+  margin-block-start: 0em;
+  margin-block-end: 0em;
+  margin-inline-start: 0px;
+  margin-inline-end: 0px;
+  padding-inline-start: 0px;
+}
+
+.calendar_item {
+  text-align: center;
+  font-size: 24px;
+  text-decoration: none;
+}
+
+.calendar_item_link {
+  text-decoration: none;
+  color: aliceblue;
+}
+
+.calendar_item_link:hover {
+  transition: text-decoration 0.5s;
+  font-weight: bolder;
+}
+
+.calendar_item_link:active {
+  font-weight: bolder;
+  color: darkturquoise;
+}
+
+.calendar_item_link-choisen {
+  font-weight: bolder;
+  color: darkturquoise;
+}
+
+.btn_prevDay {
+  text-align: center;
+  text-decoration: none;
+  color: aliceblue;
+}
+
+.btn_prevDay:active {
+  color: darkturquoise;
+}
+
+.btn_prevDay:hover {
+  border: 1px solid;
+}
+
+.btn_nextDay {
+  text-align: center;
+  text-decoration: none;
+  color: aliceblue;
+}
+
+.btn_nextDay:active {
+  color: darkturquoise;
+}
+
+.btn_nextDay:hover {
+  border: 1px solid;
+}
+
+.time_wrap {
+  display: grid;
+  grid-template-rows: repeat(2, 1fr);
+  background-color: rgba(0, 0, 0, 0.6);
+  order: 2;
+}
+
+.time_wrap_title {
+  font-size: 24px;
+  font-weight: bolder;
+  padding-left: 2em;
+  color: aliceblue;
+  letter-spacing: 4px;
+}
+
+.time_wrap_inner {
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  align-items: center;
+  list-style-type: none;
+  margin-block-start: 0em;
+  margin-block-end: 0em;
+  padding-inline-start: 0px;
+}
+
+.time_item {
+  text-align: center;
+}
+
+.time_item a {
+  font-size: 24px;
+  text-decoration: none;
+  color: aliceblue;
+}
+
+.time_item a:hover {
+  border: 1px solid;
+  border-color: darkturquoise;
+}
+
+.time_item a:active {
+  color: darkturquoise;
+}
+
+.hall_status {
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  align-items: center;
+  order: 5;
+}
+
+.seats_item {
+  position: relative;
+  font-size: 22px;
+  background-color: rgb(0, 0, 0);
+  color: white;
+  margin-block-start: 0em;
+  margin-block-end: 0em;
+  padding: 40px;
+  text-align: center;
+}
+
+.seats_item::before {
+  content: " ";
+  position: absolute;
+  width: 30px;
+  height: 30px;
+  left: 10px;
+}
+
+.seats_available::before {
+  background: grey;
+}
+
+.seats_taken::before {
+  background: rgb(240, 57, 57);
+}
+
+.seats_selected::before {
+  background: blue;
+}
+
+.seats {
+  display: grid;
+  grid-row: span 2;
+  grid-template-columns: repeat(10, 1fr);
+  min-width: 400px;
+  padding: 10px;
+  justify-self: center;
+  border: rgb(216, 208, 208) solid;
+  background-color: rgba(255, 255, 255, 0.6);
+  order: 4;
+}
+
+.seat {
+  align-self: center;
+  height: 30px;
+  width: 30px;
+  cursor: pointer;
+}
+
+.seat_input {
+  position: absolute;
+  -webkit-appearance: none;
+  -moz-appearance: none;
+  appearance: none;
+  width: 30px;
+  height: 30px;
+}
+
+.seat_number {
+  position: relative;
+  z-index: 2;
+  top: 3px;
+  left: 4px;
```

Comment:
Styles for centering elememt without relative coordinates, also, add `margin: auto` to the `.seat`:

![image](https://user-images.githubusercontent.com/16196199/107092282-f89e8780-680b-11eb-8ffd-5ba5fa9b74e1.png)